model/task/db.go

```diff
@@ -87,6 +87,9 @@ var (
 	PriorityKey                    = bsonutil.MustHaveTag(Task{}, "Priority")
 	ActivatedByKey                 = bsonutil.MustHaveTag(Task{}, "ActivatedBy")
 	StepbackDepthKey               = bsonutil.MustHaveTag(Task{}, "StepbackDepth")
+	LastFailingStepbackTaskId      = bsonutil.MustHaveTag(Task{}, "LastFailingStepbackTaskId")
```

Comment:
Should end these variable names with key, as above, to indicate that it's a bson tag key and not storing an actual ID